Foundation/NSNumber.swift

```diff
@@ -537,17 +537,20 @@ extension Bool : _ObjectiveCBridgeable {
     }
     
     public static func _forceBridgeFromObjectiveC(_ x: NSNumber, result: inout Bool?) {
-        result = _unconditionallyBridgeFromObjectiveC(x)
+        let _ = _conditionallyBridgeFromObjectiveC(x, result: &result)
```

Comment:
This isn't needed and wasn't intended to be part of the change, but the swift stdlib does something similar here, so I was testing this. I'll undo this change.

Reply:
Removed